README.md

```diff
@@ -1,4 +1,4 @@
-# Pack It! [![Build Status](https://ci.centos.org/job/packit-master/badge/icon)](https://ci.centos.org/job/packit-master)
+# Pack It! [![Build Status](https://zuul-ci.org/gated.svg)](https://softwarefactory-project.io/zuul/t/local/builds?project=packit-service/packit)
```

Comment:
It took me a while to get why the badge is static.

Reply:
:-)  https://softwarefactory-project.io/docs/zuul/user/badges.html

Reply:
Yes, I've just duckduckgo-ed that page...